src/yardstick/store/label_stats.py

```diff
@@ -0,0 +1,73 @@
+from __future__ import annotations
+
+import hashlib
+import json
+import logging
+import os
+import shutil
+from typing import Any, List
+
+from yardstick import comparison
+from yardstick.store import config as store_config
+
+LABEL_STATS_DIR = "label-stats"
+
+
+def _store_root(store_root: str | None = None) -> str:
+    if not store_root:
+        store_root = store_config.get().store_root
+    return os.path.join(store_root, LABEL_STATS_DIR)
+
+
+def store_path(ids: List[str], configuration: list[dict[str, Any]] | None, store_root: str | None = None) -> str:
+    config_str = _configuration_string(configuration)
+
+    filename = "_".join(sorted(ids)) + "_" + config_str + ".json"
```

Comment:
Does this mean the filename has a list of UUIDs in it? Any concerns that this list will be very long?

Reply:
not until it is a problem -- the vast majority of comparisons we do are 1:1 instead of 1:many. Also the label stats cache is an opt in feature that isn't used by default anywhere in the yardstick code.